### PR TITLE
added prime checker in cpp and c

### PR DESCRIPTION
**Pull request by RASHI3004**

### CODE REVIEW

**Review by OtacilioN — CHANGES_REQUESTED** [review]
Can you fix the [indentation](https://en.wikipedia.org/wiki/Indentation_style) of this file?  You can also make use of an auto formatter such as [prettier](https://prettier.io/) to improve your code style.

File: Prime Checker/prime checker c.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+
+int prime(int n)
+{
+	int i;
+  	if (n == 1) 
+	{
+    return 0;
```
Comment (OtacilioN) [inline Prime Checker/prime checker c.c:8]: Just to point one example of miss indentation we can point this line, this return should be inside the if block indentation

File: Prime Checker/prime checker.cpp [hunk 2]
```diff
@@ -5,18 +5,18 @@ int prime(int n)
...
-      		if (n % i == 0)
-			{
-        		return 0;
-      		}
-    	}
-    	return 1;
+      			if (n % i == 0)
+				{
+        				return 0;
+      				}
```
Comment (RASHI3004) [inline Prime Checker/prime checker.cpp]: I have made the changes in the indentation. I don't know why this one is showing an extra tab here but there is no such space when the file is opened.

File: Prime Checker/prime checker c.c (hunk 1 above)
Comment (RASHI3004) [reply Prime Checker/prime checker c.c]: I have made the changes in the indentation. I don't know why in one particular line it is showing an extra tab but there is no such space when the file is opened.

File: Prime Checker/prime checker c.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	int i;
+  	if (n == 1) 
+	{
+    		return 0;
+  	}
+  	else 
+	{
+    		for (i = 2; i < n; i++) 
+		{
+      			if (n % i == 0)
```
Comment (OtacilioN) [inline Prime Checker/prime checker c.c:14]: Stills miss indented, but you are closer! Pay attention and format it again, you can do it! ✨

**Review by OtacilioN — CHANGES_REQUESTED** [review]
Stills miss indented, but you are closer! Pay attention and format it again, you can do it! ✨

File: Prime Checker/prime checker c.c (hunk 3 above)
Comment (RASHI3004) [reply Prime Checker/prime checker c.c]: done

**Review by OtacilioN — APPROVED** [review]
Congratulations @RASHI3004 you got it!

**Pull request merged**